examples_tests/35.TAA/taa.frag

```diff
@@ -61,8 +90,8 @@ void main()
 	vec2 q_uv = 0.5*(q_cs.xy/q_cs.w) + 0.5;
 	vec2 vel = TexCoords - q_uv;
 */
-	// it should sample nearest (depth) in 3x3 neighbourhood
-	vec2 vel = textureLod(uVelocityBuf, TexCoords, 0.0).xy;
+	// TODO test later wether closestFragmentOffsetIn3x3() is worth it
+	vec2 vel = textureLodOffset(uVelocityBuf, TexCoords, 0.0, closestFragmentOffsetIn3x3(TexCoords)).xy;
```

Comment:
use `texelFetch`,  offset parameter in `*Offset` in GLSL has to always be a compile-time constant.